lib/class-wp-rest-menu-items-controller.php

```diff
@@ -671,7 +671,7 @@ public function prepare_item_for_response( $post, $request ) {
 
 		if ( rest_is_field_included( 'parent', $fields ) ) {
 			// Same as post_parent, expose as integer.
```

Comment:
Should we update this inline comment as well? To avoid the same mistake in the future.

Based on this property definition, `menu_item_parent` is what we need in this case.

https://github.com/WordPress/wordpress-develop/blob/f44a99529758e9a5ada443347c5febb5b8a67b6f/src/wp-includes/nav-menu.php#L797-L800

Reply:
I think it's saying to treat the `type` of `menu_item_parent` as an `int`, the same as `post_parent`, so seems correct, but maybe a bit ambiguous.